### PR TITLE
Fix missing gnupg package issue during installing k8s gpg file process

### PR DESCRIPTION
**Pull request by lixiaoyuner**
#### Why I did it
When try to install k8s gpg file, there reports an error.
- "E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation   curl: (23) Failed writing body". 

To fix error we need to install gunpg package. But apt-key add is going to be deprecated, mv the gpg file to /etc/apt/trusted.gpg.d/ directly
##### Work item tracking
- Microsoft ADO **(number only)**: 26901718

#### How I did it
Download k8s gpg file and put it in /etc/apt/trusted.gpg.d/
#### How to verify it
We can install k8s gpg file successfully, but no gnupg package installed inside sonic image.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by qiluo-msft — DISMISSED** [review]

**Comment by qiluo-msft** [comment]
@saiarcot895 Could you also help review?

File: build_debian.sh [hunk 1]
```diff
@@ -271,9 +271,11 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSIO
 
 install_kubernetes () {
     local ver="$1"
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install gnupg
     sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -fsSL \
         https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
         sudo LANG=C chroot $FILESYSTEM_ROOT apt-key add -
```
Comment (saiarcot895) [inline build_debian.sh:277]: `apt-key` is deprecated and may be removed soon. The suggestion is to just copy the file that is downloaded to `/etc/apt/trusted.gpg.d/`. (Note that you may need to change the suffix from `.gpg` to `.asc`, depending on the actual key format that is used.)

Comment (saiarcot895) [reply build_debian.sh:277]: This will also remove the gnupg dependency.

Comment (lixiaoyuner) [reply build_debian.sh]: Thanks, very helpful, changed the code as your comments.

**Review by saiarcot895 — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**